google/auth/transport/_mtls_helper.py

```diff
@@ -85,6 +87,90 @@ def _read_dca_metadata_file(metadata_path):
     return metadata
 
 
+def _get_workload_x509(certificate_config_path=None):
+    """Read the cert and key files specified in the certificate config provided.
+    If no config path is provided, check the environment variable and well known
+    gcloud location.
+
+    Args:
+        certificate_config_path (string): The certificate config path. If no path is provided,
+        the well known gcloud location will be used.
+
+    Returns:
+        Tuple[bytes, bytes]: client certificate bytes in PEM format and key
+            bytes in PEM format.
+
+    Raises:
+        google.auth.exceptions.ClientCertError: if problems occurs when retrieving
+        the certificate or key information.
+    """
+    absolute_path = _get_cert_config_path(certificate_config_path)
+    if absolute_path is None:
+        return None, None
+    data = _read_json_file(certificate_config_path)
+
+    workload = data["cert_configs"]["workload"]
+
+    cert_path = workload["cert_path"]
+    key_path = workload["key_path"]
+
+    return _read_cert_and_key_files(cert_path, key_path)
+
+
+def _get_cert_config_path(certificate_config_path=None):
+    """Gets the certificate configuration full path. If an override is provided,
```

Comment:
Current comment suggests that if override is provided and does not exist, we will fall back to the other 2 options, but the code actually returns "None" in this case. Let's update the comment for clarity:
"""Gets the certificate configuration full path using the following order of precedence:
1. Explicit override, if set
2. Environment variable, if set
3. Well-known location

Returns "None" if the selected config file does not exist.
...